Sources/General/ZLCameraConfiguration.swift

```diff
@@ -110,6 +110,9 @@ public class ZLCameraConfiguration: NSObject {
         }
     }
     
+    /// Overlay image view to be displayed on top of the camera view.
+    public var overlayImageView: UIImageView?
```

Comment:
Sorry for replying you so late. Thank you for your PR and contribution to this repository. Here are some suggestions for this submission.

Since you want to add a custom view on the custom camera, instead of defining it as `UIImageView`, it is better to change it to `UIView`, which is more flexible and users can do more operations on `UIView`.

For example, if the user does not want the frame of the added picture to be equal to the `previewFrame`. 
If it is implemented with `UIImageView`, then the user needs to make a corresponding picture.
But if using `UIView`, the user can add a `UIImageView` on `UIView`, and then use frame or layout to control the display area of ​​the picture.

If using `UIView`, it is necessary to set `isUserInteractionEnabled` to `false`. Please also reflect it in the comments.

```swift
public var overlayView: UIView? {
    didSet {
        overlayView.isUserInteractionEnabled = false
    }
}
```

Reply:
Hey @longitachi, no worries, no problems. :)
Yes, I know that `UIView` is a parent of `UIImageView`. :) However, I also wanted to match it to the other overlays in the custom camera, and since those image overlays will most likely be used 90% of the time (because for anything else, like buttons, etc., you will still need access to the custom camera props, etc.). But sure, I will change it as you suggested in about 12-14 hours. Thanks a lot!

Reply:
Hi @tsinis, thanks again for contributing to this repository. Feel free to keep mentioning PR if you have new generalized features afterward.